recipes/libpng/all/conanfile.py

```diff
@@ -173,19 +178,21 @@ def package(self):
         rmdir(self, os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.set_property("cmake_find_mode", "both")
-        self.cpp_info.set_property("cmake_file_name", "PNG")
-        self.cpp_info.set_property("cmake_target_name", "PNG::PNG")
-        self.cpp_info.set_property("pkg_config_name", "libpng") # TODO: we should also create libpng16.pc file
+        major_min_version = f"{Version(self.version).major}{Version(self.version).minor}"
 
+        # TODO: Remove after Conan 2.0
         self.cpp_info.names["cmake_find_package"] = "PNG"
         self.cpp_info.names["cmake_find_package_multi"] = "PNG"
+        self.cpp_info.names["pkg_config_name"] = ["libpng"]
```

Comment:
```suggestion
```

Must not be a list, and can be removed anyway since legacy `pkg_config` generator listens `pkg_config_name` property.